src/main/java/io/nats/client/impl/ErrorListenerLoggerImpl.java

```diff
@@ -119,4 +94,12 @@ public void pullStatusError(Connection conn, JetStreamSubscription sub, Status s
     public void flowControlProcessed(Connection conn, JetStreamSubscription sub, String id, FlowControlSource source) {
         LOGGER.info(() -> supplyMessage("flowControlProcessed", conn, null, sub, "FlowControlSource:", source));
     }
+
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public void socketWriteTimeout(Connection conn) {
+        LOGGER.severe(() -> supplyMessage("socketWriteTimeout", conn, null, null));
+    }
```

Comment:
Implementation specific implementation since the interface has a no-op default implementation.